src/preferencesControl.ts

```diff
@@ -187,23 +190,20 @@ export class PreferencesControl {
                         else {   // categValue === 'reject'
                             this.cookieCategoriesPreferences[categId] = false;
                         }
-    
-                        // Enable "Save changes" and "Reset all" buttons
+
+                        // Enable "Save changes" button
                         if (oldCategValue !== this.cookieCategoriesPreferences[categId]) {
-                            if (modalButtonSave) {
+                            if (modalButtonSave.disabled) {
```

Comment:
Is it necessary to check? If it would be `false` then setting to false one more time wouldn't hurt.